sycl/include/sycl/ext/oneapi/bindless_images_mem_handle.hpp

```diff
@@ -8,13 +8,15 @@
 
 #pragma once
 
+#include "ur_api.h"
+
 namespace sycl {
 inline namespace _V1 {
 namespace ext::oneapi::experimental {
 /// Opaque image memory handle type
 struct image_mem_handle {
-  using raw_handle_type = void *;
-  raw_handle_type raw_handle;
+  using handle_type = ur_exp_image_mem_native_handle_t;
+  handle_type raw_handle;
```

Comment:
Why the name change from `raw_handle_type` to `handle_type`? This actually differs from what we have in our spec:

```
struct image_mem_handle {
  using raw_handle_type = /* implementation defined */;
  raw_handle_type raw_handle;
}
```

Reply:
Raw handles are supposed to map to native handles, correct?

Reply:
Correct - they also need to be backend independent, which I suppose is what `ur_exp_image_mem_native_handle_t` addresses? But the names shouldn't change, otherwise we'd need to update the Bindless Images spec and update the revision.

Reply:
> which I suppose is what ur_exp_image_mem_native_handle_t addresses? 

That's right. I'll fix this name change.